src/status_im/subs/profile.cljs

```diff
@@ -41,7 +41,7 @@
    (let [{:keys [images ens-name? customization-color] :as profile} (get profiles target-key-uid)
          image-name                                                 (-> images first :type)
          override-ring?                                             (when ens-name? false)]
-     (when profile
+     (when (and profile port)
```

Comment:
Here we're checking that we have a media-server port before attempting to fetch the profile images at the login screen.